torch/autograd/functional.py

```diff
@@ -598,7 +679,7 @@ def vjp(grad_output):
 
         return _tuple_postprocess(jacobian, (is_outputs_tuple, is_inputs_tuple))
 
-def hessian(func, inputs, create_graph=False, strict=False, vectorize=False):
+def hessian(func, inputs, create_graph=False, strict=False, vectorize=False, use_forward_ad=False):
```

Comment:
You probably want to name this something more descriptive. If I see `hessian(..., use_forward_ad=True)`, then I might assume that we're using forward-mode AD for both jacobian computations. Maybe "use_forward_ad_for_outer_jacobian"? Or "outer_jacobian_strategy = {forward, reverse}"